config/config.go

```diff
@@ -100,7 +100,8 @@ type Configuration struct {
 	// Refers to main.go `configFileName` constant
 	BidderInfos BidderInfos `mapstructure:"adapters"`
 	// Hooks provides a way to specify hook execution plan for specific endpoints and stages
-	Hooks Hooks `mapstructure:"hooks"`
+	Hooks       Hooks       `mapstructure:"hooks"`
+	Validations Validations `mapstructure:"validations"`
```

Comment:
Given that these validations only validate the bid response, unless we plan to include more validations in this field in the future, should we rename to `bidResponseValidations`?

Reply:
II think we should keep the name `Validations` as that is what was specified in the issue along with what’s present in the Java version